src/vendor/Path.ts

```diff
@@ -104,7 +104,7 @@ export default class Path {
   /// rationale: usually if you are trying to join an absolute path it is a bug in your code
   /// TODO should warn tho
   join(...components: string[]): Path {
-    const joined = components.join("/")
+    const joined = components.filter(x => x).join("/")
```

Comment:
`.filter(x => x)` differ from `.compact()` for certain truthinesses? that would be very JavaScript-y.

Reply:
I forgot we provided compact lol

Reply:
oh we can't use it since Path is meant to become its own library.